index.html

```diff
@@ -6,13 +6,246 @@
+                    <source srcset='./photo/jp2/example-3.jp2' type="image/jp2"> <!-- safari -->
+                    <source srcset='./photo/webP/example-3.webp' type="image/webp"> <!-- chrome -->
+                    <img class='portfolio__item-business' src="./photo/png/example-2.png" alt="example3" loading='lazy'> <!-- default -->
+                </picture>
+                <picture class='portfolio__item-cars'>
+                    <source srcset='./photo/jp2/example-4.jp2' type="image/jp2"> <!-- safari -->
+                    <source srcset='./photo/webP/example-4.webp' type="image/webp"> <!-- chrome -->
+                    <img class='portfolio__item-cars' src="./photo/png/example-4.png" alt="example4" loading='lazy'> <!-- default -->
+                </picture>
+                <picture class='portfolio__item-design'>
+                    <source srcset='./photo/jp2/example-5.jp2' type="image/jp2"> <!-- safari -->
+                    <source srcset='./photo/webP/example-5.webp' type="image/webp"> <!-- chrome -->
+                    <img class='portfolio__item-design' src="./photo/png/example-5.png" alt="example5" loading='lazy'> <!-- default -->
+                </picture>
+                <picture class='portfolio__item-business'>
+                    <source srcset='./photo/jp2/example-6.jp2' type="image/jp2"> <!-- safari -->
+                    <source srcset='./photo/webP/example-6.webp' type="image/webp"> <!-- chrome -->
+                    <img  class='portfolio__item-business' src="./photo/png/example-6.png" alt="example6" loading='lazy'> <!-- default -->
+                </picture>
+            </div>
+        </div>
+     </section>
+     <section class='contact' id='contact'>
+                    <h2 class='heading'>
+                        contact me
+                    </h2>
+                  
+                    <form action='' class='form'>
+                 
+                            <div class='form__group'>
+                                <input type='text' class='form__input' placeholder='Full Name' id='name' autocomplete="off" required>
+                                <label for='name' class='form__label'>Full Name</label>
+                                <input type='email' class='form__input' placeholder='Email adress' id='email' autocomplete="off" required>
+                                <label for='name' class='form__label'>Email adress</label>
+                            </div>
+                            <div class='form__group'>
+                                <input type='text' class='form__input' placeholder='Subject' id='subject' autocomplete="off">
+                                <label for='subject' class='form__label'>Subject</label>
+                            </div>
+                        
+                            <textarea rows=12 class='form__textarea' placeholder='Your Meassage Here....' id='description' autocomplete="off" required>
```

Comment:
Strasznie dziwnie działa rozszerzenie tego textarea. Nie wiem jak to osiągnąłeś, ale dobrze byłoby to poprawić :) 
![image](https://user-images.githubusercontent.com/18637116/102723928-12434880-430c-11eb-8df8-eb69b205b627.png)